test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -185,84 +145,102 @@ void fillRandomTest(std::function<void(json_spirit::mValue&, bool)> _doTests, st
-	options.setWeight(dev::eth::Instruction::SSTORE, 70);
-	options.setWeight(dev::eth::Instruction::CALL, 75);
-	options.setWeight(dev::eth::Instruction::CALLCODE, 55);
-	options.addAddress(dev::Address("0xffffffffffffffffffffffffffffffffffffffff"));
-	options.addAddress(dev::Address("0x1000000000000000000000000000000000000000"));
-	options.addAddress(dev::Address("0x095e7baea6a6c7c4c2dfeb977efac326af552d87"));
-	options.addAddress(dev::Address("0x945304eb96065b2a98b57a48a06ae28d285a71b5"));
-	options.addAddress(dev::Address("0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"));
-	options.addAddress(dev::Address("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6"));
-	options.addAddress(dev::Address("0x0000000000000000000000000000000000000001"));
-	options.addAddress(dev::Address("0x0000000000000000000000000000000000000002"));
-	options.addAddress(dev::Address("0x0000000000000000000000000000000000000003"));
-	options.addAddress(dev::Address("0x0000000000000000000000000000000000000004"));
-	options.smartCodeProbability = 60;
-
+	dev::test::RandomCodeOptions options; //use default options
 	std::vector<std::string> types = getTypes();
 
 	for (std::map<std::string, std::string>::const_iterator it = _varMap.begin(); it != _varMap.end(); it++)
 		types.push_back(it->first);
 
-	for (unsigned i = 0; i < types.size(); i++)
+	for (auto const& type: types)
 	{
-		std::size_t pos = _test.find(types.at(i));
+		std::size_t pos = _test.find(type);
 		while (pos != std::string::npos)
 		{
-			if (types.at(i) == "[RLP]")
+			std::string replace;
+			if (type == "[RLP]")
 			{
 				std::string debug;
 				int randomDepth = 1 + (int)dev::test::RandomCode::randomUniInt() % 10;
-				_test.replace(pos, 5, dev::test::RandomCode::rndRLPSequence(randomDepth, debug));
+				replace = dev::test::RandomCode::rndRLPSequence(randomDepth, debug);
 				cnote << debug;
 			}
-			else
-			if (types.at(i) == "[CODE]")
-				_test.replace(pos, 6, "0x"+dev::test::RandomCode::generate(10, options));
-			else
-			if (types.at(i) == "[HEX]")
-				_test.replace(pos, 5, dev::test::RandomCode::randomUniIntHex());
-			else
-			if (types.at(i) == "[HEX32]")
-				_test.replace(pos, 7, dev::test::RandomCode::randomUniIntHex(std::numeric_limits<uint32_t>::max()));
-			else
-			if (types.at(i) == "[GASLIMIT]")
-				_test.replace(pos, 10, dev::test::RandomCode::randomUniIntHex(dev::u256("3000000000")));
-			else
-			if (types.at(i) == "[HASH20]")
-				_test.replace(pos, 8, dev::test::RandomCode::rndByteSequence(20));
-			else
-			if (types.at(i) == "[0xHASH32]")
-				_test.replace(pos, 10, "0x" + dev::test::RandomCode::rndByteSequence(32));
-			else
-			if (types.at(i) == "[HASH32]")
-				_test.replace(pos, 8, dev::test::RandomCode::rndByteSequence(32));
-			else
-			if (types.at(i) == "[V]")
+			else if (type == "[CODE]")
 			{
-				int random = (int)dev::test::RandomCode::randomUniInt() % 100;
+				int random = dev::test::RandomCode::randomPercent();
```

Comment:
A lot of `dev::test::RandomCode::`, maybe it's time for `using namespace dev::test::RandomCode;` ?